test/test_reductions.py

```diff
@@ -667,7 +667,7 @@ def test_amax(self, device, dtype):
 
     @onlyOnCPUAndCUDA
     @dtypesIfCPU(torch.float, torch.double)
-    @dtypesIfCUDA(torch.half, torch.float)
+    @dtypesIfCUDA(torch.half, torch.float, torch.bfloat16)
```

Comment:
you should also extend the OpInfo tests for `aminmax` here: https://github.com/pytorch/pytorch/blob/7f1b672b7a1c4ea2b5a71db888047feff22d04af/torch/testing/_internal/common_methods_invocations.py#L6542-L6547 by updating `dtypesIfCUDA` list to include `torch.bfloat16`:

`dtypesIfCUDA=all_types_and(torch.bool, torch.float16, torch.bfloat16),`

Reply:
Done